commands/@.js

```diff
@@ -8,88 +8,123 @@
   keyboard: 
   aliases: 
 CMD*/
-
-function isAdmin(){
+function isAdmin() {
   var admin_id = AdminPanel.getFieldValue({
     panel_name: "AdminInfo", // panel name
     field_name: "ADMIN_ID" // field name
   })
   return user.id == admin_id
 }
 
-function isNumeric(value){
+function canMasterRemove(value) {
+  if (value > 0) {
+    return false
+  }
+  var pvalue = -value
```

Comment:
why pvalue?

why not just value? Why do you need negative valiue here?
Just use value

Reply:
I really confused with this. Please use only value. pvalue is not needed

Reply:
because amount will be in negative.
it is removeable system
so there is condition of must less than 0

so value will be negative.

so converted into p value i.e. positive value